docs/reference_guides/model_libraries/generic/costing/process_costing_sslw.rst

```diff
@@ -0,0 +1,912 @@
+.. contents:: Contents 
+    :depth: 4
+
+Introduction
+------------
+
+.. rubric:: IDAES Process Costing Framework
+
+.. note:: This document outlines an example process costing library to support IDAES unit models. For a general overview of costing capabilities in IDAES, see the :ref:`IDAES Process Costing Framework<reference_guides/core/costing/costing_framework:IDAES Process Costing Framework>`.
+
+.. note:: Process costing methods are available for most of the unit operations in the IDAES Unit Model Library (Compressor, CSTR, Flash, Heater, HeatExchanger, HeatExchangerNTU, PFR, PressureChanger, Pump, StoichiometricReactor, Turbine, and generic vessels).
+
+A process costing module has been developed primarily based on base cost and purchase cost correlations 
+from the following reference with some exceptions (noted in the documentation as appropiate):
+
+Process and Product Design Principles: Synthesis, Analysis, and Evaluation. Seider, Seader, Lewin, Windagdo, 3rd Ed. John Wiley and Sons. Chapter 22. Cost Accounting and Capital Cost Estimation
+
+This library implements some of the methods described in the source, and outlines capital cost methodology for generic IDAES unit operations. The module employs Pyomo currency units using a basis Chemical Engineering Cost Index value of 500 (USD CE500) - this index creates a relative comparison scale for capital costs betweens years. Users may then pass a desired year (default USD 2018 for the SSLW costing module) and the framework will automatically convert currency units using a built-in dictionary of year/cost index definitions. The base costing framework provides a method to automatically register currency units for the costing block based on Chemical Engineering (CE) Cost Index conversion rates for US Dollars. IDAES will check if Pyomo has standard currency units registered, and if not will load unit conversions between USD at CE indices of 500 and 394, as well as USD annually from 1990 to 2020. Users should refer to the reference above for details of the costing correlations, however, a summary of the methods is provided below.
+
+Available IDAES Process Costing Module Methods
+----------------------------------------------
+
+When generating a costing block `self.costing` for a unit model `self` (e.g. `m.fs.H101.costing`), the costing block is automatically populated with the following variables:
+
+Table 1. Main Variables added to the unit block ("self.costing").
+
+=========================== ============================== ======== ==========================================================================
+Variable                    Symbol                         Units    Notes
+=========================== ============================== ======== ==========================================================================
+Purchase cost               :math:`purchase\_cost`         dollars  Purchase cost
+Base cost per unit          :math:`base\_cost\_per\_unit`  unitless Base cost per unit
```

Comment:
No need to repeat the variable names in the notes column. I'd suggest leaving it blank or even better explaining what purchase cost and base cost are and or the differences between them